{"title": "Applying the neutral indel model across metazoan genomes", "tags": ["talk", "rcg2008"]}
CREATED: 200810200457
Speaker: Chris Ponting

Gerton Lunter's indel model
* consider length of inter-gap segments, under netural model should follow geometric distribution
* at most 0.09% of all AR are under selection

Overrepresentation of long inter-gap distances
* purifying selection removes delerious indels
* 1.56-2.25% of human genome are functional but non-coding

How much protein coding regions?
* compare against mouse genomes
* about 1.06%

Are non coding transcripts functional?
* many of the 3122 nc transcripts exhibit purifying selection

NIM work well for many paris, except for closely related primates
* due to PolyA microsatellites in ALUs at 150bp apart
* departure at low distances probably due to errors in sequencing/assembly

10% of human genome appears to be constrainted by indel purifying selection
* ref: The functional repertoires of metazoan genomes, Nature Review Genetics
* much turn over in functional sequence, functional but not conserved!
