{"title": "Conservation and Evolution of Regulatory Elements in Mammalian Genomes", "tags": ["open_problem", "talk"]}
CREATED: 200703270931
Speaker: Guillaume Bourque

** ChIP-PET
 * against ChIP on Chip which uses micro array instead of sequencing to get tags
 * enrichment ~10% of fragments
 * locate PET clusters to find target sites
 * enrichment of motifs near binding regions
 * clear signals of conserved motifs across multiple species
 ** conserved motifs (specific to TF) more specific than just conserved elements
 * many of the binding sites are not conserved! (up to 50%)
 ** conserved binding sites are more meaningful
 * proportion of conserved binding sites independent of distance from gene
 * looking for other motifs in these conserved regions
 ** cofactors etc.
 * binding sites in regions with repeats
 ** repeats are highly dynamic in recent evolution
 ** different TF appears in different types of repeats
 ** ChIP-PET is able to detect repeats but ChIP on Chip is biased away from repeats

