{"title": "Lemoine2007", "tags": ["paper", "evolution", "positional_orthologs", "prokaryotes", "synteny"]}
CREATED: 201004072013
LINK: <url:~/Modules/Literature/Lemoine2007.pdf>
TITLE: Assessing the evolutionary rate of positional orthologous genes in
prokaryotes using synteny data

Proposed two strategies to extract from a set of homologs, a collection of valid
orthologs shared by at least two genomes.

First method is Reciprocal Smallest Distance (RSD) using PAM distances
separating pairs of homologs.

Second method groups homologs in families, compute gene trees, distinguishing
bona fide orthologs and paralogs created after the last speciation event.

Use orthologs generated by either method or their intersection to find
orthologs that are adjacent for each pair of genomes (Positional Orthologous
Genes).

BLAST searches often return as the highest scoring hit a protein that is not
the nearest phylogenetic neigbour. Hence actual orthologs will be missed by
RBH/BBH approaches. RSD is based on estimates of evolutionary distace rather
than BLAST scores.

Show that POGs are subject to more evolutionary constraints than orthologs
outside synteny groups, whatever the taxonomic distance separating the
compared organisms.
