{"title": "Mystery of intron splicing", "tags": ["talk", "rcg2008"]}
CREATED: 200810200719
** Paramecium (ciliate)
 * macro nucleus, somatic genome
 * micro nucleus, germline
 * programmed genome rearrangement
 * model eukaryotic unicellular organism
 * ~39642 annotated genes, 24052 (2 copies, from WGD), 11451 (1 copy)
 * 51% of ancestral genes present in 2 copies
 * DNA-DNA alignment (L Spearling)
 * 3 successive WGD! but 2x2x2 = 2
** Lifespan of gene duplicates
 * evolution of gene function (neo/sub functionalization)
 * dosage constraints (gene balance hypothesis, Papp et al 2003)
 * EST data, highly expressed genes more frequently retained
 * expression -> retention? Yes, but not as predicted by models of gene retention
 * cost of expression model
 ** energetic cost: ATP, GTP
 ** mobilizing gene expression machinery
 * highly expressed genes are more dosage sensitive
** Mystery of intron splicing
 * weak signals for splice sites/branch points
 * ~2.3 introns/gene, short ~25bp
 * deficient in introns of length = 3n
 * selective pressure to have stop codon within 3n introns? Yes
 ** splicing is error prone
 ** retention of stop codon to stop production of aberrant mRNA
 ** 3n -> has stop codon
 ** non-3n -> introduce frame shift! May produce stop codons later in exons etc
 ** NMD machinery to remove aberrant mRNA
 * deficient in 3n stopless introns in intron rich eukaryote
 * Jaillon et al. Nature 2008
