{"title": "Phylogenetic Networks", "tags": ["daniel_huson", "talk"]}

Phylogenetic network: describes evolution that includes reticulate events

Types of networks: unrooted/rooted, abstract/explicit, softwired/hardwired, combine/transfer

Unrooted/rooted: direction of time

# Unrooted

Split network to show conflicting trees (abstract)

Notion of compatibility of splits

Neighbor-net algorithm

18S rRNA sequences used for gene trees.

Consensus split network from splits that occur in X/N trees. X > N/2 => splits are always compatible.

Super split network for trees on different subset of species

Median network for alignment of binary strings.

Quasi median network is an extension to multistate character strings.

Median-joining algorithm combined Quasi-Median network and minimum spanning network. Highly cited work.

# Rooted
tree edges, reticulate edges

Each tree edges defines a cluster (the leaves of the subtree)

Clusters -> Hasse diagram -> Hardwired cluster network

Hybridization network (Dendroscope program)

Gene tree/Species tree reconciliation minimize DLT

# Links

[http://www.phylogenetic-networks.org]

[http://www.dendroscope.org]

[http://www.splitstree.org]

