{"title": "Synteny, Orthology and Evolution of Gene Clusters", "tags": ["talk"]}

Presented by Peter Stadler at GCM2009

# Hox gene clusters
Hox genes are homeodomain transcription factors

Position related to temporal and position expression of hox genes

Regulatory cascade of hox genes

Fragmentation of hox clusters in Invertebrates

ParaHox clusters

Minor change in spatial expression of Hox genes can lead to large phenotypic differences (fins vs arms)

# SynBlast
tBlastn to find homologs

most genomes are not well annotated

# Gene duplication via Co-Phylogeny
reconcillation of gene tree and species tree to infer speciation and duplication events

Co-phylogeny : reconcillation of host species tree and parasite species tree

difficult to get good gene trees

# Evolutionary patterns of ncRNA
selection pressure on secondary struture, leads to compensatory mutations

even annotating known families is challenging

Rfam: 607 known families of ncRNAs

fragrap: efficient homology search for ncRNA

- more flexible than HMM/covariance models

- still too sensitive for short ncRNAs and too specific for long ones

Syntenology (religious/ad-hoc use of synteny information)

for vtRNA in protocadherin cluster

- vertebrate procadherin cluster (associated with brain development)

- vertebrate Y RNA cluster, Y RNA gene has several copies in co-localized gene cluster
