{"title": "Translational discoveries in multiple myeloma", "tags": ["talk"]}
CREATED: 200801220205
Speaker: A/P Chng Wee Joo, Dept of Medicine

** Background
 * Hallmarks of cancers
 * Complex molecular circuitry of a cancer cell, redundancies and crosstalk
 * Linear studies of single pathway/single gene do not show the whole picture
 * Make use of global studies using microarray
 * Applications: class discovery, response prediction, marker identification
 * Multiple myeloma, malignancy of B-cells -> abnormal copy number, translocation
** Identification of hyper diploid myeloma
 * three colors per slide, can identify 3 chromosomes
 * use chromosome 9, 11, 15 to identify H-MM
 * easier to target cancer cells which are multiplying vs those which are dormant
 * heterogeneity in H-MM patients based on clustering of expression profile
 * Array-CGH to look for genetic basis of clusters
** Centrosome amplification in MM
 * centrosomes involved in dividing the chromosomes in cell division
 * prognostic factor which predicts survival rates independent of treatment
 * analysis of four genes associated with centrosomes produces an index which can be used clinically
** Integrated Pan-omics Analysis
 * input: DNA, RNA, proteins, TMA cytospin
 * technology
 ** aCGH -> genomic gain/loss
 ** GEP -> transcriptional signature
 ** proteonomic (MS, Ab Array) -> protein signature
 * analysis: pathway, GO, GSEA
 * output
 ** Etiology (genetic basis of molecular phenotype)
 ** diagnosis and prognosis (clinically relevant subtypes and their molecular markers)
 ** therapy (deregulated pathways)
 * validation
