{"title": "Wolfe1997", "tags": ["paper", "genome_duplication", "yeast"]}
CREATED: 200906190356
LINK: <url:~/Modules/Literature/Wolfe1997.pdf>
Data: yeast genome

** Results
Identified 55 duplicate regions, containing 376 pairs of homologous genes

AA sequence identity between pairs ranges from 24% to 100% with a mean of 63%

** Definition of duplicate region
BLASTP high scores of >= 200 for each gene pair

At least  three pairs of homologous with intergenic distances of <= 50kb on each chromosome

Conservation of gene order and orientation (with allowance for small inversions within some blocks)

** Details of regions
on average 55kb long, contain a mean of 6.9 duplicate gene pairs

span 50% of the genome

in 55 duplicate regions, 25% (743 of 2905) genes are duplicated

the 376 pairs of duplicate proteins account for 12% of all  yeast sequence
pairs having a BLASTP score above 200

these 376 pairs account for 42% of all "simple" duplicate yeast pairs in yeast
(pairs of genes that are one another's only significant BLASTP hit)

** Two possible explanations
many successive independent duplications (each involving several kilobases)

tetraploidy followed by reciprocal translocation between chromosomes to
produce a mosaic of duplicated blocks

** Results favor the tetraploidy and translocation model
for 50 of the 55 duplicate regions, orientation of the entire block wrt the
centromere is the same in the two copies, which is significantly non
random based on chi-square tests

for simple gene pairs in which both genes lie outiside of the 55 blocks, most
(117 out of 172 pairs) show conserved transcriptional orientation, whereas
no bias of orientation is seen in pairs in which one of the genes lies in
the block but the other does not

based on Poisson distribution of block sizes, 55 successive duplications would
be expected to result in about seven triplicated regions but we observe none
(or at most one)

comparison with Kluyveromyces suggests that these lineages diverge before the
proposed genome duplication in yeast, this implies that the Kluyveromyces
homologs of all the genes found in the duplicate blocks should be single-copy

model of massive gene deletion in the wake of genome duplication predicts some
groups of genes that are adjacent in Klu to have homologs in yeast that are
not themselves duplicated but are located within different copies of
duplicated blocks. Followed by examples of such.

for most of the 376 gene pairs, the function of both genes is not known, only
a few pairs have functions that have clearly diverged.
